### PR TITLE
[IMP] mrp,stock: change icon for sml button

### PR DESCRIPTION
**Pull request by svs-odoo**
Task #1966567

### CODE REVIEW

File: addons/mrp/views/mrp_production_views.xml [hunk 1]
```diff
@@ -62,7 +62,7 @@
                                 <span class="o_stat_text">Work Orders</span>
                             </div>
                         </button>
-                        <button name="%(action_mrp_production_moves)d" type="action" string="Inventory Moves" class="oe_stat_button" icon="fa-arrows-v" attrs="{'invisible': [('state', 'not in', ('progress', 'done'))]}"/>
+                        <button name="%(action_mrp_production_moves)d" type="action" string="Inventory Moves" class="oe_stat_button" icon="fa-exchange" attrs="{'invisible': [('state', 'not in', ('progress', 'done'))]}"/>
```
Comment (lap-odoo) [inline addons/mrp/views/mrp_production_views.xml]: @svs-odoo  Replace "Inventory Moves" by "Product Moves" please

File: addons/stock/views/stock_quant_views.xml [hunk 2]
```diff
@@ -40,7 +40,7 @@
             <form string="Inventory Valuation" create="false" edit="false" delete="false">
                 <sheet>
                     <div class="oe_button_box" name="button_box">
-                        <button class="oe_stat_button" icon="fa-arrows-v" type="object" name="action_view_stock_moves" string="Picking Moves"/>
+                        <button class="oe_stat_button" icon="fa-exchange" type="object" name="action_view_stock_moves" string="Picking Moves"/>
```
Comment (lap-odoo) [inline addons/stock/views/stock_quant_views.xml]: @svs-odoo Rename "Picking Moves" into "Product Moves"

File: addons/mrp/views/mrp_unbuild_views.xml [hunk 3]
```diff
@@ -108,7 +108,7 @@
                     <sheet>
                         <div class="oe_button_box" name="button_box">
                             <button class="oe_stat_button" name="%(action_mrp_unbuild_moves)d"
-                                    string="Product Moves" type="action" icon="fa-arrows-v" states="done"/>
+                                    string="Product Moves" type="action" icon="fa-exchange" states="done"/>
```
Comment (lap-odoo) [inline addons/mrp/views/mrp_unbuild_views.xml]: @svs-odoo it sends back to the stock.moves, I'd prefer to show the stock.move.lines. 
@sle-odoo What do you think of it ?

Comment (sle-odoo) [reply addons/mrp/views/mrp_unbuild_views.xml]: @lap-odoo yes it would be more consistent with the other modules

**Comment by sle-odoo** [comment]
robodoo r+

**Pull request closed without merging**